about/faq.rst

```diff
@@ -38,6 +38,8 @@ Which platforms are supported by Godot?
 * Windows
 * macOS
 * X11 (Linux, \*BSD)
+* Web (Chrome 68 or later, Firefox 79 or later, Edge 79 or later), see: :ref:`using the web editor <doc_using_the_web_editor>`)
```

Comment:
I wouldn't mention browser versions as these are bound to change in the future. Also, Opera should work as it's Chromium-derived:

```suggestion
* Web (Chrome, Firefox, Edge, Opera), see: :ref:`using the web editor <doc_using_the_web_editor>`)
```

I don't remember if Safari works with the web editor.